external/storm-kafka/src/test/storm/kafka/KafkaUtilsTest.java

```diff
@@ -168,7 +168,7 @@ public void generateTuplesWithValueSchemeAndKeyValueMessage() {
     }
 
     private ByteBufferMessageSet getLastMessage() {
-        long offsetOfLastMessage = KafkaUtils.getOffset(simpleConsumer, config.topic, 0, OffsetRequest.LatestTime()) - 1;
+        long offsetOfLastMessage = KafkaUtils.getOffset(simpleConsumer, config.topic, 0, OffsetRequest.LatestTime());
```

Comment:
This change seems a little odd to me.  Do we know why the tests off-by-one after this change?